### PR TITLE
update contrib/README.md

### PR DESCRIPTION
**Pull request by mburdette**

### CODE REVIEW

File: contrib/README.md [hunk 1]
```diff
@@ -2,20 +2,29 @@
...
 # run bump_version.bash with either patch, minor, or major
 ./bump_version.bash <patch|minor|major>
 
-# make live
-git push upstream && git push upstream --tags
-npm publish
+# push changes to your remote origin 
+git push origin release/3.<x>.<y>
 
-npm deprecate mongodb-stitch "The browser SDK has moved to mongodb-stitch-browser-sdk, and the Node.js SDK has moved to mongodb-stitch-server-sdk"
```
Comment (UnicodeSnowman) [inline contrib/README.md:16]: this `npm deprecate` call still needs to be run after release, so I would just move this after the `npm publish` step

File: contrib/README.md [hunk 2]
```diff
@@ -2,20 +2,29 @@
 
 ### Summary
 
-This project follows [Semantic Versioning 2.0](https://semver.org/). In general, every release is associated with a tag and a changelog. `master` serves as the mainline branch for the project and represent the latest state of development.
+This project follows [Semantic Versioning 2.0](https://semver.org/). In general, every release is associated with a tag and a changelog. `support/3.x` serves as the mainline branch for the project and represent the latest state of development.
```
Comment (UnicodeSnowman) [inline contrib/README.md:5]: [nit] ... and represent**s** the latest...

File: contrib/README.md [hunk 3]
```diff
@@ -2,20 +2,29 @@
...
 ./bump_version.bash <patch|minor|major>
 
-# make live
-git push upstream && git push upstream --tags
-npm publish
+# push changes to your remote origin 
+git push origin release/3.<x>.<y>
 
-npm deprecate mongodb-stitch "The browser SDK has moved to mongodb-stitch-browser-sdk, and the Node.js SDK has moved to mongodb-stitch-server-sdk"
+# open a PR from that remote branch to suppor/3.x and get it reviewed
```
[the remark this answers is not in the document]
Comment (mburdette) [reply contrib/README.md:22]: ✔️

File: contrib/README.md (hunk 1 above)
Comment (mburdette) [reply contrib/README.md:16]: woops

**Pull request merged**